packages/react/src/schematics/application/files/css-module/src/app/__fileName__.module.__style__

```diff
@@ -0,0 +1,128 @@
+.app {
+  font-family: sans-serif;
+  min-width: 300px;
+  max-width: 600px;
+  margin: 50px auto;
+}
+
+.app .gutter-left {
+  margin-left: 9px;
+}
+
+.app .col-span-2 {
+  grid-column: span 2;
+}
+
+.app .flex {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.app header {
+  background-color: #143055;
+  color: white;
+  padding: 5px;
+  border-radius: 3px;
+}
+
+.app main {
+  padding: 0 36px;
+}
+
+.app p {
+  text-align: center;
+}
+
+.app h1 {
+  text-align: center;
+  margin-left: 18px;
+  font-size: 24px;
+}
+
+.app h2 {
+  text-align: center;
+  font-size: 20px;
+  margin: 40px 0 10px 0;
+}
+
+.app .resources {
+  text-align: center;
+  list-style: none;
+  padding: 0;
+  display: grid;
+  grid-gap: 9px;
+  grid-template-columns: 1fr 1fr;
+}
+
+.app .resource {
```

Comment:
This won't work since `.resource` will be converted to a hash so you cannot do `className="resource"`. 

Change it to just `.resource { ... }` and then in the JSX use  `className={styles.resource}`.

Reply:
Alternatively, you can do this:

```
.app :global(.resource) {
  // ...
}
```

Which will make `resource` a global class name rather than a local (encapsulated) one.